pilot/pkg/security/authn/v1alpha1/policy_applier.go

```diff
@@ -168,12 +162,8 @@ func convertToEnvoyJwtConfig(policyJwts []*authn_v1alpha1.Jwt) *envoy_jwt.JwtAut
 					},
 				},
 				Requires: &envoy_jwt.JwtRequirement{
-					RequiresType: &envoy_jwt.JwtRequirement_RequiresAny{
-						RequiresAny: &envoy_jwt.JwtRequirementOrList{
-							Requirements: append(requirements, &envoy_jwt.JwtRequirement{
-								RequiresType: &envoy_jwt.JwtRequirement_AllowMissingOrFailed{},
-							}),
-						},
+					RequiresType: &envoy_jwt.JwtRequirement_AllowMissingOrFailed{
```

Comment:
from envoy proto comment for ```allow_missing_or_failed``` - 
```
// The requirement is always satisfied even if JWT is missing or the JWT
// verification fails.
```
with this change, does it mean jwt filter won't fail the request even jwt missing/invalid ?

Reply:
yes and actually it's so even without this change or with the old code for istio jwt filter.